libraries.json

```diff
@@ -753,5 +753,10 @@
     "github": "corbella83/PopKorn",
     "category": "DI",
     "maven": "https://repo1.maven.org/maven2/cc/popkorn/popkorn/"
+  },
+  {
+    "github": "ln12/multiplatform-connectivity-status",
+    "category": "Utils",
+    "maven": "https://github.com/ln-12/multiplatform-connectivity-status"
```

Comment:
here should be path to maven repo directory with `maven-metadata.xml` which contains versions.
see here for example - https://repo1.maven.org/maven2/cc/popkorn/popkorn/

if library not published to public maven script can't fetch info and will not add library to list

Reply:
I uploaded it to maven central